OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -173,7 +185,7 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Mo
 				return true;
 			};
 
-			chatDisabledLabel = new CachedTransform<int, string>(x => x > 0 ? $"Chat available in {x} seconds..." : "Chat Disabled");
+			chatDisabledLabel = new CachedTransform<int, string>(x => modData.Translation.GetString(ChatAvailability, Translation.Arguments("seconds", x)));
```

Comment:
Here is an example of `CachedTransform` use that may help for my comments above.